packages/accordion/package.json

```diff
@@ -27,13 +27,13 @@
     "exports": {
         ".": "./src/index.js",
         "./src/*": "./src/*",
-        "./custom-elements.json": "./custom-elements.json",
         "./package.json": "./package.json",
         "./sp-accordion": "./sp-accordion.js",
         "./sp-accordion.js": "./sp-accordion.js",
         "./sp-accordion-item": "./sp-accordion-item.js",
         "./sp-accordion-item.js": "./sp-accordion-item.js"
     },
+    "customElementsManifest": "custom-elements.json",
```

Comment:
Will this still be in the package when this is released? We're using custom-elements.json to construct some linting rules.

Reply:
This will still be in the package. 

However, talk to me about these lint rules...they sound exciting! While I am interested in whether we could surface them more broadly, tell me specifically about how they are built. It _is_ possible that certain tooling will cease to find them if they are listed this way.

Depending on what that looks like, I can also see removing this one commit from the PR to prevent having to worry about any complication here.

Reply:
I _think_ there should be no issue here in the linting code we've looked at so far. However, to support proving that, I've moved this change into its own PR. This way we can work through all of the currently open PRs, and then do a test release with the `custom-elements.json` change, and depending on how that effects these lint rules move forward with a full release on one side or the other of that commit. Sound reasonable?

Reply:
Definitely -- although in this case you might be doing more work than necessary. I just wanted to be sure the file would still exist -- I feel like I might have inadvertently led you down more of a rabbit hole than necessary.